src/main/java/uk/gov/mca/beacons/service/dto/AccountHolderDTO.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.mca.beacons.service.dto;
+
+public class AccountHolderDTO extends DomainDTO {
+
+  private String type = "accountHolder";
+
+  public String getType() {
+    return type;
+  }
```

Comment:
We tried inlining this like:

```java
  public String getType() {
    return "accountHolder";
  }
```

But it was causing `400 BAD REQUEST` errors (request body not parsed).  @issyjenkins and I couldn't figure out why!!! Please help @matthew-a-carr ?

Reply:
If it's a 400 then it's presumably something to do with Spring failing to deserialise a key from the JSON object it receives. 

I image it's due to the fact that if you do how you have done in the above, i.e: returning the accountHolder string in the function then the DTO object is not going to have a `type` key.

I would have to dig into the error messages that Spring throws to determine the root cause, but my hunch is that it's something to do with that `type` property not existing is why it fails.